sunpy/util/tests/test_datatype_factory_base.py

```diff
@@ -63,91 +63,84 @@ def different_validation_function(cls, *args, **kwargs):
         return kwargs.get('style') == 'missing-different'
 
 
-class TestBasicRegistrationFactory:
+def test_default_factory():
```

Comment:
Also haven't checked this properley, but looks fine at a glance.

Reply:
Same here, no changes other than de-indent + remove `self` from function arguments.